ikasaneip/cli/shell/bin/ikasan-config-service.sh

```diff
@@ -53,9 +53,9 @@ if [[ -f "$LOCAL_ENV" ]]; then
     . ${LOCAL_ENV}
 fi
 
-CONFIG_SERVICE_URL=${CONFIG_SERVICE_URL:-'http://localhost:8880'}
-CONFIG_SERVICE_BOOTSTRAP_LOCATION=${CONFIG_SERVICE_BOOTSTRAP_LOCATION:-'~/bootstrap.properties'}
+CONFIG_SERVICE_URL=${CONFIG_SERVICE_URL:-"http://localhost:8880"}
+CONFIG_SERVICE_BOOTSTRAP_LOCATION=${CONFIG_SERVICE_BOOTSTRAP_LOCATION:-"$HOME/bootstrap.properties"}
```

Comment:
Why do Linux bash scripts need the quotes?

Reply:
using single quotes it was putting the single quotes in the command line and not picking up the bootstrap.properties, using double quotes fixed this.